src/app/features/event/power-icon/power-icon.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["power-icon.scss","power-icon.css"],"names":[],"mappings":"AAAA;EACE,iBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,kBAAA;EACA,aAAA;EACA,uBAAA;EACA,mBAAA;ACCF;ADCE;EACE,iBAAA;ACCJ;ADEE;EACE,UAAA;EACA,WAAA;EACA,kCAAA;EACA,gBAAA;EACA,aAAA;EACA,sBAAA;ACAJ;ADEI;EACE,kBAAA;EACA,SAAA;EACA,QAAA;ACAN;ADGI;EACE,kBAAA;ACDN;ADII;EACE,kBAAA;EACA,QAAA;ACFN;ADKE;EACE,aAAA;EACA,uBAAA;EACA,mBAAA;EACA,iBAAA;ACHJ;ADKE;EACE,iBAAA;ACHJ;;ADOA;EACE,iBAAA;ACJF;;ADOA;EACE,eAAA;ACJF;;ADOA;EACE,iBAAA;ACJF;;ADOA;EACE,iBAAA;ACJF;;ADOA;EACE,kBAAA;EACA,QAAA;EACA,iBAAA;ACJF;;ADOA;EACE,kBAAA;EACA,QAAA;EACA,iBAAA;ACJF;;ADOA;EACE,kBAAA;EACA,QAAA;EACA,iBAAA;ACJF;;ADOA;EACE,kBAAA;EACA,eAAA;ACJF;;ADOA;EACE,oCAAA;EACA,qCAAA;ACJF;ADKE;EACE,4BAAA;ACHJ;;ADOA;EACE,6BAAA;EACA,8BAAA;ACJF;ADKE;EACE,4BAAA;ACHJ;;ADOA;EACE,oCAAA;EACA,qCAAA;ACJF;ADKE;EACE,0BAAA;ACHJ;;ADOA;EACE,0NAAA;EAGA,sFAAA;ACNF;;ADSA;EACE,0NAAA;EAGA,sFAAA;ACRF;;ADWA;EACE,0NAAA;EAGA,sFAAA;EACA,cAAA;ACVF","file":"power-icon.css"}
```

Comment:
co to za plik?

Reply:
Włączyłem przypadkowo wtyczkę Sass auto compiler, ona wygenerowała te pliki. Usunąłem